app/client/cli/cmd.go

```diff
@@ -0,0 +1,34 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/pokt-network/pocket/runtime/test_artifacts"
+	"github.com/spf13/cobra"
+)
+
+const cliExecutableName = "client"
+
+var (
+	remoteCLIURL       string
+	privateKeyFilePath string
+)
+
+func init() {
+	rootCmd.PersistentFlags().StringVar(&remoteCLIURL, "remote_cli_url", test_artifacts.DefaultRemoteCLIURL, "takes a remote endpoint in the form of <protocol>://<host> (uses RPC Port)")
```

Comment:
I was thinking... Shall we move defaults under a specific package? Maybe still under `runtime` but explicitly calling it `defaults`. 

Something like `"github.com/pokt-network/pocket/runtime/defaults"`? It feels weird to have references to test_artifacts here in a main.go

Reply:
+1 Let's do it

Reply:
https://github.com/pokt-network/pocket/pull/177/commits/aa1c7378b3970831c19cb589a496d1ac4d499818